.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Description
```

Comment:
I think this is covered by https://github.com/sous-chefs/.github/tree/master/.github/ISSUE_TEMPLATE